counterpartylib/lib/blocks.py

```diff
@@ -339,6 +339,14 @@ def initialise(db):
         cursor.execute('''INSERT INTO assets VALUES (?,?,?)''', ('0', 'BTC', None))
         cursor.execute('''INSERT INTO assets VALUES (?,?,?)''', ('1', 'XCP', None))
 
+    # Add asset_longname for sub-assets
```

Comment:
following the same logic used for `blocks` (https://github.com/CounterpartyXCP/counterparty-lib/blob/master/counterpartylib/lib/blocks.py#L202), also add `asset_longname` to the `CREATE TABLE IF NOT EXISTS assets(` statement at the end